sys/dev/netmap/netmap_vale.c

```diff
@@ -752,8 +751,7 @@ netmap_get_bdg_na(struct nmreq *nmr, struct netmap_adapter **na,
 	if (!create)
 		return ENXIO;
 	/* yes we should, see if we have space to attach entries */
-	needed = 2; /* in some cases we only need 1 */
-	if (b->bdg_active_ports + needed >= NM_BDG_MAXPORTS) {
+	if (b->bdg_active_ports >= NM_BDG_MAXPORTS) {
```

Comment:
The previous check was too conservative: ```>=``` should have been ```>``` and ```needed``` should have been computed more precisely.

Your change, however, is wrong. There _are_ cases when we need two slots. See where ```cand2``` is actually used.

Reply:
hello how are you me it is natacha I am unmarried Canadian here to make me friends here is my skype natacha dipou sorry for the inconvenience thank you and have a good day as you